react/react_and_the_backend/project_messaging_app.md

```diff
@@ -0,0 +1,19 @@
+### Introduction 
+
+Think of your favorite messaging app - Discord, Telegram, WhatsApp, Facebook Messenger, etc. - and imagine what it would be like if you could build your own. This project will give you a chance to do just that. You'll be building a web app that allows users to send messages to each other. 
```

Comment:
I kind of want to slip in an AOL Instant Messenger or MSN Messenger reference here because I'm old and nostalgic 😆 

```suggestion
Think of your favorite messaging app - Discord, Telegram, WhatsApp, Facebook Messenger, AOL Instant Messenger, etc. - and imagine what it would be like if you could build your own. This project will give you a chance to do just that. You'll be building a web app that allows users to send messages to each other. 
```